scripts/git/fix_copybara_merge.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fixes a Copybara push that failed to create a merge commit, using the
+# COPYBARA_TAG label to add a second parent to the HEAD commit.
+# This is mostly intended to be used by automation.
+
+set -e
+
+COPYBARA_TAG="COPYBARA_INTEGRATE_REVIEW"
+UPSTREAM_REMOTE="${UPSTREAM_REMOTE:-upstream}"
+
+if [[ -n "$(git status --porcelain)" ]]; then
+  echo "Working directory not clean. Aborting"
+  git status
+  exit 1
+fi
+
+if [[ -n "$(git rev-list --merges HEAD^..HEAD)" ]]; then
+  echo "HEAD commit is already a merge commit. Aborting"
```

Comment:
Would it be helpful to echo the git log graph before/after the changes made by this script?

```
git log --left-right --graph --oneline --boundary google...main-to-google
```

(changing the branch names as needed)

Reply:
This is a bit trickier because the commit we're merging from isn't necessarily the tip of main, which I'd also like to show, and then symmetric differences get all confused. Figured out a way to do it, I think. git log is unhelpful about highlighting a specific commit.